sdk/tables/azure-data-tables/tests/test_table.py

```diff
@@ -383,7 +383,6 @@ def test_set_table_acl_too_many_ids(self, tables_storage_account_name, tables_pr
         finally:
             ts.delete_table(table.table_name)
 
-    @pytest.mark.live_test_only
```

Comment:
These used to be live-only to prevent the sas tokens being in the recordings. Are they being substituted out of recordings?

Reply:
Yes they are now.